govuk-app-linter.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# set -e
+
+ENVIRONMENTS=("integration" "staging" "production")
+
+for env in "${ENVIRONMENTS[@]}"; do
+  array_length=$(yq e ".govukApplications | length - 1" "./charts/app-config/values-$env.yaml")
+
+  if [ "$array_length" -le 0 ] ; then
+    continue;
+  fi
+
+  echo "=================== $env ===================";
+
+  declare -i errors=0;
+
+  # expand all the aliases
+  yq 'explode(.)' "./charts/app-config/values-$env.yaml" > "lint-tmp/$env-exploded.yaml"
+
+  for app_index in $(seq 0 "$array_length");do
+    filename="./lint-tmp/$(yq e ".govukApplications[$app_index].name" "./charts/app-config/values-$env.yaml")-chart.yaml"
+
+    echo "$app_index/$array_length: linting $filename";
+
+    yq e ".govukApplications[$app_index].helmValues" "lint-tmp/$env-exploded.yaml" > "$filename";
+
+    output=$(helm lint --quiet -f "$filename" ./charts/generic-govuk-app)
```

Comment:
Bare in mind there are apps in the list which use different charts, like asset-manager

Reply:
Maybe something like this
```suggestion
    CHART_PATH=$(yq -r '.chartPath // "charts/generic-govuk-app"' "${filename}")
    output=$(helm lint --quiet -f "$filename" ${CHART_PATH)
```

Reply:
I'll try that out

Reply:
Testing it out, doesn't quite work so I'm fixing it